### DISCUSSION TITLE
magick.exe silent with powershell

### DISCUSSION THREAD

**Discussion by ZeekExile** [ImageMagick/ImageMagick#7717] [Help]
### ImageMagick version

ImageMagick-7.1.1-39

### Operating system, version and so on

Win x64

### Description

I have a simple script in PowerShell to bulk-convert the heic files 

` # Run mogrify to batch convert HEIC files to JPG in the destination folder
    $arguments = "mogrify -format jpg -path `"$destinationFolder`" -quiet `"$sourceFolder\*.heic`""

    # Execute the command using Start-Process
    Start-Process -FilePath $magickPath -ArgumentList $arguments -NoNewWindow -Wait`

All is working but when I run it I get a magick.exe window popping up. Is there way to run it silently?

**Comment by snibgo** [comment]
I suggest using "`magick mogrify`" instead of just "`mogrify`".

What happens if you replace the ImageMagick command with some other program, such as:
```
xcopy nosuch.exe x.txt
```
Does the problem still occur? If it does, then the problem is not specific to IM. I don't use Powershell, and can't advise.

**Reply by ZeekExile** [reply]
it uses magick.exe its described under variable $magickPath
the mogrify is part of the argument. 

Interestingly when place the script on the network drive it is no longer popping so I think I have managed to sort this
